runelite-client/src/main/java/net/runelite/client/menus/MenuManager.java

```diff
@@ -282,6 +282,14 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 		eventBus.post(playerMenuOptionClicked);
 	}
 
+	private String removeTagsFromPlayerMenuTarget(String target)
```

Comment:
I don't think it warrants a seperate function since you are only adding a single line. I think a comment would be better.